docs/_specification/1.2-DRAFT/metadata.md

```diff
@@ -105,7 +105,7 @@ See the appendix [RO-Crate JSON-LD](appendix/jsonld) for details.
 
 ## Additional metadata standards
 
-RO-Crate also uses the _Portland Common Data Model_ ([PCDM] version <https://pcdm.org/2016/04/18/models>) to describe repositories or collections of digital objects and imports these terms:
```

Comment:
this version also appears in context.jsonld for both 1.1 and 1.2 - I've raised https://github.com/duraspace/pcdm/issues/83 to see if this can be fixed, as the pcdm.org site is under new ownership

Reply:
The versioned link is now back up - I'm going to revert the change to this line, then I'll merge the PR